package.json

```
@@ -4,9 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build:client": "webpack --config webpack.client.config.js",
+    "build:client": "if exist dist rmdir /s /q dist && webpack --config webpack.client.config.js",
```

macOS 환경에서 `npm run start`를 실행했을 때 콘솔에 아래와 같은 문법 오류가 발생합니다.

```Bash
> react-ssr@1.0.0 build:client
> if exist dist rmdir /s /q dist && webpack --config webpack.client.config.js

sh: -c: line 1: syntax error: unexpected end of file
```

크론이 올려주신 뼈대 코드의 원래 수정 내용은 아래와 같아요. 여기 작성된 내용을 보니 윈도우 환경에서의 명령어인 것 같은데, 혹시 아래와 같은 코드가 동작하지 않아서 이렇게 수정하신 것인지 궁금합니다.

```suggestion
    "build:client": "rm -rf dist && webpack --config webpack.client.config.js",
```

---

앗 이 부분을 pr에 남긴다는 걸 깜빡했네요! 
크론이 알려주신 뼈대 코드로는 윈도우 환경에서는 작동하지 않아서 수정하게 되었어요.